packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -425,6 +427,9 @@ const DataGridProVirtualScroller = React.forwardRef<
         <GridVirtualScrollerRenderZone {...getRenderZoneProps()}>
           {mainRows}
         </GridVirtualScrollerRenderZone>
+        <GridVirtualScrollerRenderZone {...getFocusRenderZoneProps()}>
```

Comment:
We might consider similar solution for columns virtualization, since we have the same problem with horizontal scrolling